docs/distributions/fedora/installation.md

```diff
@@ -21,7 +21,7 @@ If you need a more updated kernel, use the iso from [sharpenedblade](https://git
 
 ## Installing unsupported spins
 
-1. Follow the installation instructions above, but use your custom (vanilla) ISO. You need an external keyboard and mouse. WiFi will not work until you finish the installation procedure so you need a wired internet connection.
+1. Follow the installation instructions above, but use your custom (vanilla) ISO. You need an external keyboard and mouse. If you do not have a wired internet connection, you need to follow the [Wi-Fi guide](https://wiki.t2linux.org/guides/wifi-bluetooth/) before proceding.
```

Comment:
Probably mention that you have to follow the Wi-Fi guide in the live iso environment.